node/core/approval-voting/src/approval_db/v1/mod.rs

```diff
@@ -25,10 +25,11 @@ use polkadot_primitives::v1::{
 use sp_consensus_slots::Slot;
 use parity_scale_codec::{Encode, Decode};
 
-use std::collections::{BTreeMap, HashMap};
-use std::collections::hash_map::Entry;
+use std::collections::BTreeMap;
 use bitvec::{vec::BitVec, order::Lsb0 as BitOrderLsb0};
 
+pub(crate) const STORED_BLOCKS_KEY: &[u8] = b"Approvals_StoredBlocks";
```

Comment:
This also seems like something that should be constrained to this module and not public.